conformance/lang/expressions/let-expr/let_expr.balt

```diff
@@ -0,0 +1,481 @@
+Test-Case: output
+Description: Test let-expr, when type-descriptor is a nil-type-descriptor which is a simple-type-descriptor.
```

Comment:
Do we really need " which is a simple-type-descriptor."?

Reply:
Removed